client/my-sites/marketplace/types.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { YOAST_PREMIUM, YOAST_FREE } from '@automattic/calypso-products';
+
+export interface IProductDefinition {
+	/**
+	 * defaultPluginSlug : Some plugins may not be available in the calypso product library so we specify a default plugin that is accessible in the product library
+	 * so that relevant details can be shown
+	 * */
+	defaultPluginSlug: string;
+	pluginsToBeInstalled: string[];
+	isPurchasableProduct: boolean;
+}
+
+export interface IProductCollection {
```

Comment:
`IProductCollection` seems pretty specific to be an interface. Is there a reason why it's not just an object type?

Reply:
I am still learning how to balance typescript. All to remove that pesky red/yellow lines when coding. 😄 
This can actually be helpful though to link the user to the products supported.